OpenAI-DotNet-Tests/TestFixture_03_Chat.cs

```diff
@@ -678,5 +679,30 @@ public async Task Test_11_GetChatVisionStreaming()
             Console.WriteLine($"{result.FirstChoice.Message.Role}: {result.FirstChoice.Message.Content} | Finish Reason: {result.FirstChoice.FinishDetails}");
             result.GetUsage();
         }
+
+        [Test]
+        public async Task Test_12_ReadRateLimitHeaders()
```

Comment:
I don't think we need a new test for this. Just add logging to the one of the existing tests